### DISCUSSION TITLE
Attempted import error: 'react-loading-skeleton' does not contain a default export (imported as 'Skeleton')

### DISCUSSION THREAD

**Discussion by agkruchkov** [dvtng/react-loading-skeleton#146] [Q&A]
**Describe the bug**
When compiling i get error:
Attempted import error: 'react-loading-skeleton' does not contain a default export (imported as 'Skeleton')

**To Reproduce**
1) installed
npm i react-loading-skeleton

2) I wrote in component: 

import Skeleton from 'react-loading-skeleton';
import 'react-loading-skeleton/dist/skeleton.css';

**Versions**

-   react-loading-skeleton version: 3.1.0
-   Browser version: Microsoft Edge, ver 104.0.1293.63

What could be the problem? Help me please.

**Comment by srmagura** [comment]
Hello, this is not enough information to reproduce the problem. Please provide a runnable example that shows the error (a GitHub repository or a CodeSandbox).

**Comment by agkruchkov** [comment]
Unfortunately, I can't grant access to the working project.

And I see that there is an export inside the library. But we don't have this problem with other libraries.

**Reply by srmagura** [reply]
Can you create a small create-react-app project that demonstrates the issue? Or copy your real project and delete all of the real code except the react-loading-skeleton import?

**Reply by agkruchkov** [reply]
![rsl1](https://user-images.githubusercontent.com/61491839/188182306-696ec167-3f10-441a-aa22-50e7d19cb7ab.png)
![rsl2](https://user-images.githubusercontent.com/61491839/188182310-98de5532-9853-4cfa-802b-0e67863ca718.png)

hmm...Webpack for some reasons does not want to understand export?..

**Reply by srmagura** [reply]
That is very odd. Can you post your `tsconfig.json`? And, if this is a create-react-app project, what is the version of `react-scripts`?

**Comment by agkruchkov** [comment]
"react-scripts": "2.1.1",

tsconfig.json:
{
  "compilerOptions": {
    "outDir": "build/dist",
    "module": "es6",
    "target": "es5",
    "lib": ["es7", "es2017", "dom", "esnext.array"],
    "experimentalDecorators": true,
    "sourceMap": true,
    "allowJs": true,
    "jsx": "react",
    "moduleResolution": "node",
    "rootDirs": [
      "src",
      "../"
    ],
    "forceConsistentCasingInFileNames": true,
    "allowSyntheticDefaultImports": true,
    "baseUrl": "./src/",
    "noErrorTruncation": true,
    "paths": {
      "common/*" : ["../../common/*"]
    },
    "esModuleInterop": true
  },
  "exclude": [
    "../node_modules",
    "build",
    "../.history",
    "scripts",
    "config",
    "acceptance-tests",
    "webpack",
    "jest",
    "src/setupTests.ts",
    "app-features"
  ],
  "types": [
    "typePatches",
    "node"
  ]
}

**Comment by agkruchkov** [comment]
I tried updating the libraries react-scripts, ts-loader, typescript.
Don't help.

**Reply by srmagura** [reply]
Hmm, I can't say what is going wrong. react-scripts v2 is very old so that could be the problem, but you said you tried updating it. I may be able to help if you provide a runnable example that shows the problem.

**Reply by agkruchkov** [reply]
I can't provide such a project.
Anyway, thanks a lot for trying to help!
If a problem is found, I'll be sure to post the solution here.

**Comment by benjp009** [comment]
Did you try running `npm install react-loading-skeleton` in the app terminal ? 
That fixed it for me
